src/app/app.component.ts

```diff
@@ -126,8 +126,23 @@ export class MyApp implements OnInit, AfterViewInit {
       this.handleBackButton();
       this.appRatingService.checkInitialDate();
       this.getUtmParameter();
+      this.checkForCodeUpdates();
     });
   }
+  checkForCodeUpdates() {
+    SunbirdSdk.instance.sharedPreferences.getString("deploymentKey").toPromise().then(deploymentKey => {
```

Comment:
@sharathkashyap  We can use preferences ( defined in the constructor) instead of  SunbirdSdk.instance.sharedPreferences.

Reply:
I have made appropriate changes